talentlink-connector/processes/rest/Attachment.p.json

```diff
@@ -0,0 +1,81 @@
+{
+  "format" : "10.0.0",
+  "id" : "1882E31A11B7E690",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "rest.AttachmentData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "getAttachment(Long)",
+      "config" : {
+        "callSignature" : "getAttachment",
+        "input" : {
+          "params" : [
+            { "name" : "attachmentId", "type" : "Long" }
+          ],
+          "map" : {
+            "out.attachmentId" : "param.attachmentId"
+          }
+        },
+        "result" : {
+          "params" : [
+            { "name" : "attachment", "type" : "com.axonivy.connector.talentlink.entity.Attachment" },
+            { "name" : "error", "type" : "ch.ivyteam.ivy.bpm.error.BpmError" }
+          ],
+          "map" : {
+            "result.attachment" : "in.#attachment",
+            "result.error" : "in.#error"
+          }
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 64 }
+      },
+      "connect" : { "id" : "f4", "to" : "f5" }
+    }, {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 352, "y" : 64 }
+      }
+    }, {
+      "id" : "f5",
+      "type" : "RestClientCall",
+      "name" : "Attachment",
+      "config" : {
+        "path" : "/candidate/attachment/{documentId}",
```

Comment:
there is a good testing-infrastructure available that covers the existing connector functionality: 
I thin you can easily adapt and extend it, to cover this attachment service call too.
See for instance https://github.com/axonivy-market/talentLink-connector/blob/master/talentlink-connector-test/src_test/com/axonivy/connector/talentlink/connector/test/CandidateTest.java

Reply:
if you get a 404 , I think you just have to enrich the existing mock-service with a method that describes the 'attachment' resource: https://github.com/axonivy-market/talentLink-connector/blob/master/talentlink-connector-test/src/com/axonivy/connector/talentlink/connector/test/TalentlinkServiceMock.java
... but I leave it up to you and the guys that invented this connector @LukasKulhanek + @juergen-plohn , whether it's worth the effort to cover this resource with a test.